notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1986 @@
+{
```

Comment:
<p>Text says latest version of GCPC, but you have the version pinned</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Nice catch, fixed.</p>